### DISCUSSION TITLE
Gschema not working in 100%

### DISCUSSION THREAD

**Discussion by Bammede** [blue-build/.github#42] [Support] [answered]
I have setup gschema from here https://blue-build.org/reference/modules/gschema-overrides/ and  it works for some entries, other entries are ignored.

1. Ptyxis Gnome Terminal config is applied for all users except Ptyxis theme aka palette.

This works:
```
[org.gnome.Ptyxis]
restore-session=false
restore-window-size=false
font-name='Cascadia Code 14'
use-system-font=false
default-profile-uuid='171d3479003fcda3fe68d39e66d229e0'
profile-uuids=['171d3479003fcda3fe68d39e66d229e0']
```
This is ignored:
```
[org.gnome.Ptyxis.Profiles.171d3479003fcda3fe68d39e66d229e0]
label='Default'
palette='dracula'
```
apss folders settings are ignored too:

```
[org.gnome.desktop.app-folders]
folder-children=['02c8ef21-504a-4191-a08f-fa02e6d3024b', '81aa131d-84f8-4559-9f1c-a64aa56ed712', '88f7d639-59e7-442b-b7be-f7dd3b153682', 'c7afdbe7-cb7b-49f2-8a63-ba89c683d24a', '7af9bdf4-e852-4978-adaf-7ce7c7a8af60', '90761dfb-3648-44b8-a1cb-a47914ccf8a3', '2d9697aa-e7da-4a52-bf18-47ffdb7b3817']

[org.gnome.desktop.app-folders.folders.2d9697aa-e7da-4a52-bf18-47ffdb7b3817]
apps=['page.kramo.Sly.desktop', 'app.fotema.Fotema.desktop', 'org.gnome.Loupe.desktop', 'io.gitlab.adhami3310.Converter.desktop']
name='Photos'

[org.gnome.desktop.app-folders.folders.7af9bdf4-e852-4978-adaf-7ce7c7a8af60]
apps=['de.leopoldluley.Clapgrep.desktop', 'com.github.huluti.Curtail.desktop', 'fr.romainvigier.MetadataCleaner.desktop', 'io.github.peazip.PeaZip.desktop', 'dev.geopjr.Collision.desktop', 'io.gitlab.adhami3310.Impression.desktop', 'mediainfo-gui.desktop', 'org.nickvision.tubeconverter.desktop', 'com.github.ADBeveridge.Raider.desktop', 'io.github.zaedus.spider.desktop', 'com.github.ryonakano.reco.desktop']
name='Tools'

.
.
.
```

**Comment by fiftydinar** [comment, marked as answer]
Those are ignored, because they are relocatable schemas, which are not supported, as outlined in docs.

Use `dconf-update-service` bling submodule for those relocatable schemas instead:
https://blue-build.org/reference/modules/bling/#dconf-update-service

**Reply by vostronix** [reply]
Hi, exist some project where i can see a example how to use the dconf-update-service with app folders ?

**Reply by fiftydinar** [reply]
> Hi, exist some project where i can see a example how to use the dconf-update-service with app folders ?

Hello, here:
https://github.com/fiftydinar/gidro-os/blob/b13135a3cfeb617aba6e5574d733e6c25c09daa6/files/0-system/etc/dconf/db/distro.d/01-gidro

**Reply by vostronix** [reply]
Thx for the replay this is helping me :) is the file name important "01-gidro" ?

**Reply by fiftydinar** [reply]
> is the file name important "01-gidro" ?

Generally no.

**Reply by vostronix** [reply]
ok can you see what i did wrong ? because i have really no clue anymore ^^ https://github.com/vostronix/my-fedora

**Reply by fiftydinar** [reply]
> ok can you see what i did wrong ? because i have really no clue anymore ^^ https://github.com/vostronix/my-fedora

Maybe it's because of spaces in `=`

**Reply by vostronix** [reply]
I changed it and is not working. How i tested it ! I using Gnome Boxes and will run this command like in the readme just with the right tag `rpm-ostree rebase ostree-image-signed:docker://ghcr.io/vostronix/my-fedora:ba5915d-41 `

**Comment by Bammede** [comment]
Resolved
